### ISSUE TITLE
Cannot launch/install app on device

### ISSUE THREAD

**Issue by PixeLInc** [bhdouglass/clickable#56]
Heyo - I can't seem to run/install the app on my device.
It goes through and pushes it to the device which I can see on my device via terminal in the main home directory but it fails to install/launch it.

Logs:
```
Successfully built package in './appname.yourname_1.0.0_all.click'.
/mnt/backup/Coding/clickable-ubuntu/app...ushed. 5.6 MB/s (15840 bytes in 0.003s)
Installing files              [=========================]         
Finished                      [=========================]         
Fatal error: could not resolve: appname.yourname_1.0.0_all.click

** (process:3418): WARNING **: Unable to connect to Upstart bus: Cannot autolaunch D-Bus without X11 $DISPLAY

** (process:3418): WARNING **: Unable to start app 'appname.yourname_appname_1.0.0': Invalid app ID: appname.yourname_appname_1.0.0
```

Furthermore, I can in fact install the click file via 
```
pkcon --allow-untrusted install-local clickfilename.click
```
on my device perfectly fine after clickable pushes said click file and run it and it works great.

Running on Arch Linux, Device is a Moto G 2014 w/ Ubuntu Touch Vivid.

**Comment by bhdouglass** [comment]
It looks like you are using one of the `clickable init` templates, did you make any major changes to the code before building it?

**Comment by bhdouglass** [comment]
Could this have anything to do with the device being the Moto G? It's my understanding that it's not an official device and the progress is still ongoing.

**Comment by PixeLInc** [comment]
That was an idea of mine but I don't know. It's quite odd because it 100% pushes the click file to the device with no issue at all it just looks like it can't find it when trying to install it.
I can potentially try and maybe spin up an emulator or something of the sort with touch installed and test it out.

**Comment by bhdouglass** [comment]
When the click gets pushed to the device where is it located? I believe it should be sent to `/home/phablet/appname.yourname_1.0.0_all.click`

**Comment by PixeLInc** [comment]
Yes, it is sent to the home directory of phablet.

**Comment by bhdouglass** [comment]
What's the default directory when you adb shell into the device? You can simply run this command: `adb shell pwd`

**Comment by PixeLInc** [comment]
Heyo - Yes Sorry, I've been a bit busy and distracted.

Running `adb shell pwd` on the device returns `/root`

**Comment by bhdouglass** [comment]
Ok, interesting, that should be `/home/phablet`. So the problem is that the clicks are being placed in `/home/phablet`, but the command to install them is being run in `/root` which explains why it can't find the click. Any idea how your home directory (or user), got changed like that?

**Comment by PixeLInc** [comment]
No, no idea. I've never messed with anything like that after installing touch on it. All I've done is install a few apps and just leave it sitting since I haven't had much use for it.

**Comment by bhdouglass** [comment]
Weird, I've made a small change in the code that should fix your problem. I'm putting together a release, so it should be available soon.

**Comment by PixeLInc** [comment]
Alrighty. Thank you :)

**Comment by bhdouglass** [comment]
This should be fixed in v4.3.0 (soon building in the PPA)
